system-contracts/bootloader/bootloader.yul

```diff
@@ -3762,6 +3736,8 @@ object "Bootloader" {
                     setNewBatch(PREV_BATCH_HASH, NEW_BATCH_TIMESTAMP, NEW_BATCH_NUMBER, EXPECTED_BASE_FEE)
                 }
 
+                GAS_PRICE_PER_PUBDATA := gasPerPubdataFromBaseFee(EXPECTED_BASE_FEE, FAIR_PUBDATA_PRICE)
```

Comment:
So, for playground batches we are not guaranteeing a minimal amount of pubdata bytes, right? What's the use case?

Reply:
The use-case is not about the pubdata price, but more about the base fee (i.e. sometimes in the API we want to simulate transaction under the base fee lower than the current one). But then we also recalculate the gas per pubdata

Reply:
But, in theory, the operator anyway can provide a low base fee. The only thing that this line allows is - a low base fee, while the pubdata price is high(`TX_GAS_LIMIT/ERG_PER_PUBDATA < GUARANTED_PUBDATA`)